From: Baruch Siach <baruch@tkos.co.il>
Subject: [PATCH] poco: don't build debug libraries

Remove the debug libraries build dependency from the default target.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---

--- poco-1.4.1p1-dist/build/rules/compile	2011-02-09 11:12:24.000000000 +0200
+++ poco-1.4.1p1/build/rules/compile	2011-08-03 07:51:42.946044176 +0300
@@ -14,7 +14,7 @@
 
 all: $(DEFAULT_TARGET)
 all_static: static_debug static_release
-all_shared: shared_debug shared_release
+all_shared: shared_release
 all_debug: static_debug shared_debug
 all_release: static_release shared_release
 
